src/auth/basic/qgsauthbasicmethod.cpp

```diff
@@ -272,6 +283,7 @@ bool QgsAuthBasicMethod::updateDataSourceUriItems( QStringList &connectionItems,
       else
       {
         connectionItems.append( caparam );
+        qWarning() << QStringLiteral( "Connection items after appending" ).arg( connectionItems.join( "&" ) );
```

Comment:
you're missing a %1 here

Reply:
It's not going to end up in the final PR and good enough to provide the required info in the travis log, no worries ;)